src/components/Channel/ChannelContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import ChannelMain from './ChannelMain';
+import ChannelSide from './ChannelSide';
```

Comment:
```suggestion
import React from 'react';
import styled from '@emotion/styled';

import ChannelMain from './ChannelMain';
import ChannelSide from './ChannelSide';
```

라이브러리와 컴포넌트 부분은 한 줄 띄워서 구분해주시면 좋을 것 같습니다